### PR TITLE
Updates to User Documentation

### PR DESCRIPTION
**Pull request by Quarriers89**
## Summary of Changes

Added updates to introduction user documentation (glossary, introduction, etc.)

Addresses [ORCA-58](https://bugs.earthdata.nasa.gov/browse/ORCA-58)

## Changes

* Created documentation for navigating ORCA docs, contributing to ORCA, the Glossary, and intro pages.

## Type of change

Please delete options that are not relevant.

- [X ] New feature (non-breaking change which adds functionality)
- [X ] This change requires a documentation update

## How Has This Been Tested?

No testing required. Documentation only. Requires review through Jira process.

## Checklist:

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [ ] CHANGELOG.md
    - [ ] Testing documentation
    - [X ] Development documentation
    - [X ] User documentation
    - [ ] (Additional here)
- [X ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
    - [ ] Unit tests
    - [ ] Integration tests
    - [ ] Manual tests (outlined above)
- [ ] New and existing unit tests pass locally with my changes
    - [ ] Automated tests passing (if not fork)
    - [ ] Manual testing/integration testing passes (if forked)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code has passed security scanning
    - [ ] Snyk
    - [ ] git-secrets

## What is the current behavior?

N/A

## What is the expected correct/added behavior?

N/A

## Relevant logs and/or screenshots

N/A

## Possible fixes

N/A

### CODE REVIEW

File: website/docs/about/introduction/orca-intro.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ ORCA Working Group.
 
 ## How do I implement or use ORCA?
 
-Some words here to talk about Cumulus integration/deployment at a very high level.
+Information on implementing ORCA is located here or in the
+[ORCA GitHub repository](https://github.com/nasa/cumulus-orca).
 
+Information on ORCA use for Operators and Data Managers is located here.
```
Comment (bhazuka) [inline website/docs/about/introduction/orca-intro.md:20]: Need to create a link on **here** to the stub out of operator documents.

**Review by bhazuka — APPROVED** [review]
Overall looks good. I added a few thoughts and ideas for changes and additions.

File: website/docs/about/introduction/contributing.md [hunk 2]
```diff
@@ -4,3 +4,59 @@ title: Contributing to ORCA
 description: Provides high level information on contributing to the ORCA project.
 ---
 # Types of Contributers
+## Engagement Levels
+ORCA has several ways to engage with the project and several different
+engagement levels. The various levels are explained below.
+
+**Core Developer:**  Part of the core ORCA team that develops code and 
+documentation for the ORCA components. If you would like to be a part of
+this team, please contact Darla Werner.
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/contributing.md:13]: Made that update. Good idea... then someone doesn't have to dig for Darla's contact info.

File: website/docs/about/introduction/contributing.md [hunk 3]
```diff
@@ -4,3 +4,59 @@ title: Contributing to ORCA
...
+
+**DAAC Integrator:**  Developers, engineers, and operators that implement
+ORCA into their DAAC workflows and Cumulus instances.
+
+**Advisor:**  Stakeholders that would like to be invited to demos and
+provide input on where the system is headed. This includes active engagement
+in the ORCA Working Group.
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/contributing.md:20]: Done

File: website/docs/about/introduction/contributing.md [hunk 4]
```diff
@@ -4,3 +4,59 @@ title: Contributing to ORCA
...
 ---
 # Types of Contributers
+## Engagement Levels
+ORCA has several ways to engage with the project and several different
+engagement levels. The various levels are explained below.
+
+**Core Developer:**  Part of the core ORCA team that develops code and 
+documentation for the ORCA components. If you would like to be a part of
+this team, please contact Darla Werner.
+
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/contributing.md:14]: Done

File: website/docs/about/introduction/contributing.md [hunk 5]
```diff
@@ -4,3 +4,59 @@ title: Contributing to ORCA
...
+Here is an example of creating a feature branch.
+
+### Creating a feature branch
+
+```bash
+git clone https://github.com/nasa/cumulus-orca.git
+
+cd cumulus-orca
+
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/contributing.md:57]: Done.

File: website/docs/about/introduction/glossary.md [hunk 6]
```diff
@@ -3,7 +3,207 @@ id: intro-glossary
...
+
+## B
+### Bucket
+An Amazon S3 cloud storage resource.
+
+For more information, see the [AWS Bucket Documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html).
+
+## C
+
+### CloudFormation
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/glossary.md:37]: Spoke with Jacob, and he recommended removing CloudFormation and CloudFormation Template entries in the glossary. Done.

File: website/docs/about/introduction/glossary.md [hunk 7]
```diff
@@ -3,7 +3,207 @@ id: intro-glossary
...
+programmatically or from the command line. For more information,
+see the [AWS IAM Documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+
+## B
+### Bucket
+An Amazon S3 cloud storage resource.
+
+For more information, see the [AWS Bucket Documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html).
+
+## C
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/glossary.md:35]: Done.

File: website/docs/about/introduction/navigating-the-docs.md [hunk 8]
```diff
@@ -3,4 +3,22 @@ id: intro-navigating
 title: Navigating the ORCA Documents
 description: Provides tips and tricks to navigate the ORCA documentation.
 ---
-# Developer Guide
+# Developer Documents
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/navigating-the-docs.md:6]: Done.

File: website/docs/about/introduction/navigating-the-docs.md [hunk 9]
```diff
@@ -3,4 +3,22 @@ id: intro-navigating
...
+### ORCA Developer Documentation
+ORCA developer documentation can be found here (TBD).
+
+ORCA README documents in the [ORCA main repository](https://github.com/nasa/cumulus-orca).
+### Cumulus Developer Documentation
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/navigating-the-docs.md:11]: Done. Added Cumulus Developer Documentation to the footer.

File: website/docs/about/introduction/orca-intro.md (hunk 1 above)
Comment (Quarriers89) [reply website/docs/about/introduction/orca-intro.md:20]: Done.

File: website/docs/about/introduction/orca-intro.md [hunk 10]
```diff
@@ -14,6 +14,8 @@ ORCA Working Group.
 
 ## How do I implement or use ORCA?
 
-Some words here to talk about Cumulus integration/deployment at a very high level.
+Information on implementing ORCA is located here or in the
+[ORCA GitHub repository](https://github.com/nasa/cumulus-orca).
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/orca-intro.md:18]: Done.

File: website/docs/about/introduction/glossary.md [hunk 11]
```diff
@@ -3,7 +3,207 @@ id: intro-glossary
...
+### ECS Activity
+An ECS instance run via a Step Function.
+### EMS
+[ESDIS Metrics System](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems)
+
+### Execution (Cumulus)
+A Cumulus execution refers to a single execution of a (Cumulus) Workflow.
+## F
+
+## G
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/glossary.md:97]: Done.

File: website/docs/about/introduction/glossary.md [hunk 12]
```diff
@@ -3,7 +3,207 @@ id: intro-glossary
...
+See [node.js](https://nodejs.org/en/about).
+### NPM
+Node package manager.
+
+For more information, see [npmjs.com](https://www.npmjs.com/).
+## O
+### Operator
+Refers to those tasked with monitoring, configuring or otherwise utilizing
+umulus in an operational deployment.
+## P
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/glossary.md:139]: Done.

File: website/docs/about/introduction/navigating-the-docs.md [hunk 13]
```diff
@@ -3,4 +3,22 @@ id: intro-navigating
 title: Navigating the ORCA Documents
 description: Provides tips and tricks to navigate the ORCA documentation.
 ---
-# Developer Guide
+# Developer Documents
+### ORCA Developer Documentation
+ORCA developer documentation can be found here (TBD).
```
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/navigating-the-docs.md:8]: Done.

File: website/docs/about/introduction/navigating-the-docs.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Quarriers89) [reply website/docs/about/introduction/navigating-the-docs.md:11]: Done

**Pull request merged**